src/api/ledger/parse/trustline.js

```diff
@@ -15,8 +15,10 @@ function parseFlag(flagsValue, trueValue, falseValue) {
   return undefined;
 }
 
-function parseQuality(quality) {
-  return (new BigNumber(quality)).shift(-9).toNumber();
+function parseQuality(quality?: number) {
+  if (typeof quality === 'number') {
```

Comment:
Good point. Rippled docs say it should be a number, but I suppose it's safer to check. Can you add the `return undefined` though?